### DISCUSSION TITLE
Issues sending mail with Postal

### DISCUSSION THREAD

**Discussion by Larkings** [postalserver/postal#1959] [Help with using Postal] [answered]
I am having this issue
I have set up an SMTP server and installed Postal

When I send a test email / my email says "Pending"

smtper.net I receive this error:

SMTP send error
Failure sending mail.
Unable to read data from the transport connection: The connection was closed.

Gmail responded:
530 Authentication required

When I run Postal Status:

root@mail:~# postal status
Name Command State Ports

postal_cron_1 /docker-entrypoint.sh post ... Up
postal_requeuer_1 /docker-entrypoint.sh post ... Exit 1
postal_smtp_1 /docker-entrypoint.sh post ... Up
postal_web_1 /docker-entrypoint.sh post ... Up
postal_worker_1 /docker-entrypoint.sh post ... Exit 1

How do I get all of these to say "up" ??
Please help..

**Comment by willpower232** [comment, marked as answer]
Have you tried `postal start` again? Its possible that either RabbitMQ or your database weren't fully started when the docker containers tried to run

**Reply by Larkings** [reply]
I have unfortunately, it keeps showing the same status.

**Reply by willpower232** [reply]
you should be able to do `postal logs worker` to see what problem it has

**Reply by Larkings** [reply]
I fixed the issue, thank you for the assistance

**Reply by egboche** [reply]
.....what's the next step after checking postal status...I mean how do I fix the problem.

**Reply by willpower232** [reply]
if all the containers are `Up` and the log output doesn't say anything relevant then it is possible the messages aren't in the queue properly and you have to clear that in the database

**Reply by Akazox** [reply]
> I fixed the issue, thank you for the assistance

how did you fix your issue ? i'm having the same

**Reply by Larkings** [reply]
I'm assuming the place you found the steps didn't fully elaborated on how to do the steps correctly. At the section where you have to do some adjustments you should also change the info in the main en the message db besides the rabbit mq. if you need me to go in dept send me a message.

**Reply by Akazox** [reply]
many tutorials are not up to date, and it's my first experience so i did a LOT of research  and i have re done the steps many time, i had issues with mariadb database localhost acces denied etc... i fixed that 
but now im having this issue and it's frustrating ngl , are you talkin about the ymal ?

**Reply by Larkings** [reply]
Indeed the yml.

**Reply by rewarduzoije** [reply]
Please how did you fix this issues? Because I too can't send email with my postal I couldn't even send a test email please I need help

**Reply by NaumanAnsari30** [reply]
> I'm assuming the place you found the steps didn't fully elaborated on how to do the steps correctly. At the section where you have to do some adjustments you should also change the info in the main en the message db besides the rabbit mq. if you need me to go in dept send me a message.

Can you please guide how you fixed this issue?
